src/client/client.cpp

```diff
@@ -286,13 +286,18 @@ void Client::draw() {
             case ObjectType::Player: {
                 // don't render yourself
                 if (this->session->getInfo().client_eid.has_value() && sharedObject->globalID == this->session->getInfo().client_eid.value()) {
-                    glm::vec3 pos = sharedObject->physics.position;
+                    /*glm::vec3 pos = sharedObject->physics.position;
+                    pos.y += PLAYER_EYE_LEVEL;*/
+                    //  TODO: Update the player eye level to an acceptable level
+                    glm::vec3 pos = sharedObject->physics.corner + (sharedObject->physics.dimensions / 2.0f);
                     pos.y += PLAYER_EYE_LEVEL;
                     cam->updatePos(pos);
                     break;
                 }
                 auto lightPos = glm::vec3(0.0f, 10.0f, 0.0f);
-                auto player_pos = glm::vec3(sharedObject->physics.position.x, sharedObject->physics.position.y + 0.1, sharedObject->physics.position.z);
+                //auto player_pos = glm::vec3(sharedObject->physics.position.x, sharedObject->physics.position.y + 0.1, sharedObject->physics.position.z);
+
+                auto player_pos = sharedObject->physics.corner + (sharedObject->physics.dimensions / 2.0f);
```

Comment:
Maybe we can make a function on the Physics struct called getCenterPos that does this calculation for us?

Reply:
Added!